strawberry_django/ordering.py

```diff
@@ -39,9 +43,18 @@ class Ordering(enum.Enum):
     DESC = "DESC"
 
 
-def generate_order_args(order: WithStrawberryObjectDefinition, prefix: str = ""):
+def generate_order_args(
+    order: WithStrawberryObjectDefinition,
+    *,
+    sequence: Optional[_OrderSequence] = None,
+    prefix: str = "",
+):
+    sequence = sequence or {}
     args = []
-    for f in order.__strawberry_definition__.fields:
+    for f in sorted(
+        order.__strawberry_definition__.fields,
+        key=lambda f: (seq := sequence.get(f.name, (0,))) and seq[0],
```

Comment:
I'd make this into a function, I'm not 100% sure what it does 😊

Reply:
I refactored it, let me know if it is better :)